packages/v4/src/content/design-guidelines/components/page/page.md

```diff
@@ -2,4 +2,58 @@
+
+The basic layout of a page differs depending on whether your application uses vertical or horizontal primary navigation as illustrated below.
+
+### Page with vertical navigation
+
+<img src="./img/page-vertical-nav.png" alt="page with vertical navigation" width="832"/>
+
+**1. Masthead:** The masthead provides a consistent header for every page in your application.
+
+**2. Navigation:** This is your primary navigation. It should be consistent across pages. See the [Navigation design guidelines](/components/navigation/design-guidelines) for more information about navigation options and usage.
+
+**3. Breadcrumb (optional):** When breadcrumbs are included, they should be the topmost element in the header section. When breadcrumbs are included on a page, the top padding is reduced to 16px. See the [Breadcrumb design guidelines](/components/breadcrumb/design-guidelines) for more information about breadcrumb usage.
+
+**4. Page header:** All pages should have a header section. The height of the header will vary to fit the content. We recommend keeping information in the header area brief to consume as little space as possible. The page header should contain the page title to help orient the user and an optional description to describe the contents of the page. You may optionally apply the `pf-m-sticky-top` modifier to the header section to create a sticky page header and prevent its content from scrolling off the page.
+
+**5. Body section:** Your main page content goes here. The body section can have a light gray `--pf-global--BackgroundColor--200` or white background depending on the content it will contain. The default padding all around the page body should be 24px in all directions.
+
+**6. Footer section (optional):** You can optionally define a footer section for a page. You may optionally apply the `pf-m-sticky-bottom` modifier to the footer section to create a sticky page footer and prevent its content from scrolling off the page.
+
+### Page with horizontal navigation
+
+<img src="./img/page-horizontal-nav.png" alt="page with horizontal navigation" width="832"/>
+
+When using a horizontal primary navigation, the left-hand sidebar is removed and the navigation is placed inline with other masthead elements. All other page elements remain the same.
+
+## Masthead
+The masthead contains global properties such as logotype, navigation and settings in an organized fashion and it is accessible from all parts of the application. All pages should share a common masthead.
+
+<img src="./img/masthead.png" alt="masthead" width="832"/>
```

Comment:
Small thing, but shouldn't the number in the notification badge be a part of the badge (ie: inside the blue box) and not outside of it?